lib/install/validate/storage.go

```diff
@@ -31,6 +31,11 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/datastore"
 )
 
+const (
+	nfsScheme = "nfs"
```

Comment:
this constant is defined several times in vic-machine, you might move that into cmd/vic-machine/common, or rely on portlayer constant directly.
maybe the latter is better, cause vic-machine will rely on portlayer anyway